packages/bz2.rb

```diff
@@ -50,6 +50,7 @@ def self.install
     system "mkdir", "-p", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}"
     system "cp", "-p", "libbz2.so.1.0.6", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}"
     system "ln", "-s", "libbz2.so.1.0.6", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}/libbz2.so.1.0"
+    system "ln", "-s", "libbz2.so.1.0", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}/libbz2.so.1"
```

Comment:
Actually, now that I saw it a second time, can you please change `#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}` to `#{CREW_DEST_LIB_PREFIX}`?

Reply:
Also, I noticed you could change `#{CREW_DEST_DIR}/usr/local` to `#{CREW_DEST_PREFIX}`.